Backpack/Icon/Classes/BPKIcon.m

```diff
@@ -125,6 +123,19 @@ + (UIImage *)iconNamed:(NSString *)name color:(UIColor *)color size:(BPKIconSize
     return icon;
 }
 
++(NSString *)codepointForIcon:(NSString *)name size:(BPKIconSize)size {
+    NSString *lookupKey = name;
+    if(size == BPKIconSizeSmall) {
+        lookupKey = [NSString stringWithFormat:@"%@%@", name, @"-sm"];
+    }
+
+    if (self.iconMapping) {
+        NSAssert([self.iconMapping objectForKey:lookupKey], @"Unknown icon `%@` does not correspond to a known icon. Please check that the icon exists at the size you are trying to use https://backpack.github.io/components/icon?platform=design", lookupKey);
+    }
+
+    return [self.iconMapping objectForKey:lookupKey];
```

Comment:
Unrelated: I noticed we aren't handling the `nil` case explicitly for the above assert. Can we verify that it does something sensible(like returning an empty image)

Reply:
Yeah this is the same as the logic we had before - will double check though 👍